wagtail/admin/templates/wagtailadmin/pages/side_panels/includes/page_locale_status.html

```diff
@@ -19,9 +19,10 @@
 
 {% block action %}
     {% if translations %}
-        <div data-button-with-dropdown>
+        <div class="w-inline-flex w-justify-center" data-button-with-dropdown>
             {% trans 'Switch locales' as action_text %}
             {% include 'wagtailadmin/pages/side_panels/includes/side_panel_button.html' with attr='data-button-with-dropdown-toggle' text=action_text %}
+            {% icon name='arrow-down' class_name='w-w-5 w-h-5 w-text-teal-200 w-transition motion-reduce:w-transition-none' %}
```

Comment:
It’s very surprising to me that this icon would be outside the button, thus not clickable, even though it’s styled to look like the button. Could we instead move the icon to be inside the `side_panel_button`?

Ideally switch to something like:

```
{% include 'wagtailadmin/pages/side_panels/includes/side_panel_button.html' with has_toggle=True text=action_text %}
```

which would add the `data-` attribute, and the icon.

Reply:
I've made this change now :) I also created a plugin for the icon rotation because then it hooks into the tippy instance and can look at when the tooltip is shown or not to rotate the icon more efficiently haha.